pending/itunes.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    BleachBit
+    Copyright (C) 2015 Andrew Ziem
+    http://bleachbit.sourceforge.net
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+    @app     iTunes
+    @link    https://www.apple.com/itunes/
+    @os      Windows
+    @version 0.1.0
+-->
+<cleaner id="itunes" os="windows">
+  <label>iTunes</label>
+  <description>Media Player</description>
+  <option id="cache">
+    <label>Cache</label>
+    <description>Delete the cache</description>
+    <action search="file" command="delete" path="$LocalAppData\Apple Computer\iTunes\Cache.db"/>
+  </option>
+  <option id="cache_artwork">
+    <label>Album Artwork Cache</label>
+    <description>Delete the cache</description>
+    <action search="walk.all" command="delete" path="$USERPROFILE\My Music\iTunes\Album Artwork\Cache"/>
```

Comment:
@az0 `$USERPROFILE\My Music` doesn't work herer. The windows explorer correctly redirects to my moved folder but bleachbit cannot follow.
I found https://github.com/az0/bleachbit/blob/96500aafb38c0539159b764a53be1fef55988762/bleachbit/Windows.py#L542 . How can I use the decleration of 'music'? `$MUSIC` and `$MYMUSIC` don't work.